samples/cmake/ozz-animation/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+if(NXDK)
+  # Hack for memory.h and friends
+  include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/")
```

Comment:
This is missing; it's probably left in https://github.com/JayFoxRox/nxdk/tree/38f909193b29b9859ff38a45ff343e9d6180233e or I forgot to add this file to my repo.

I assume it's just an empty dummy though?